core/src/main/java/org/elasticsearch/cluster/service/ClusterService.java

```diff
@@ -244,6 +240,35 @@ protected synchronized void doStop() {
     protected synchronized void doClose() {
     }
 
+    class ClusterServiceTaskBatching extends TaskBatching {
```

Comment:
Sorry to get hung up on the naming, I know it's annoying, but should it be `ClusterServiceTaskBatcher` and `TaskBatcher`?